### ISSUE TITLE
Please share BIOS settings

### ISSUE THREAD

**Issue by jgehw** [RobyRew/MSI_X570_A-Pro_Hackintosh_OpenCore#8]
First of all, many thanks @RobyRew for this guide!

Trying to clone your setup, I bought almost identical hardware: same motherboard, Ryzen 5950X (I couldn't get a Ryzen 3950X, they're out of stock, too old probably), 2x 32GB 3200MHz memory; and I re-used a Radeon 570 GPU from my old Intel Hackintosh.

I updated BIOS to the latest version `7C37vHH` (2022/05/24), loaded default values, and did these changes:
- Above 4G memory/Crypto Currency mining: Enabled
- DRAM Frequency: DDR4-3200
- Serial(COM) Port: Disabled
- Secure Boot: Disabled
- Resume By USB Device: Enabled
- Boot Order (1st UEFI USB Key, 2nd UEFI Hard Disk)

I created a bootable USB key with your `EFI` folder cloned and a HFS+ partition with the macOS installer. First OpenCore crashed before displaying the picker. The error messages pointed me to issues in the `config.plist`, and I could solve them by these changes to `config.plist`:
- convert `UEFI` / `Drivers` / `Item0` etc. from type dictionary to type string, just holding the strings `HfsPlus.efi` and `OpenRuntime.efi`
- remove `UEFI` / `Quirks` / `ForceOcWriteFlash`

(Later I noticed the reason for these errors is that your commit "Updated to OpenCore 0.7.3" includes `config.plist` but is missing the OpenCore binaries `BOOT` / `BOOTx64.efi`, `OC` / `OpenCore.efi`, and `OC` / `Drivers` / `OpenRuntime.efi` and `OpenCanopy.efi`, so instead of my changes to `config.plist` one can just update these mentioned OpenCore binaries.)

Then OpenCore booted into the picker, found the USB key partition with the installer, and started it. However, then it freezes with just the Apple logo being displayed on the screen. I found out that I can get some details displayed by adding `-v` to the `NVRAM` / `Add` / `7C436110-AB2A-4BBB-A880-FE41995C9F82` / `boot-args` string in `config.plist`. I then see (last lines before freeze):
```
...
[EB|`B:WFDW] Err(0xE), 0 @ LocHB 71B4903C-14EC-42C4-BDC6-CE1449930E49
[EB|#LOG:DT] 2022-07-07T18:18:38
[EB|#LOG:EXITBS:START] 2022-07-07T18:18:38
```
To rule out issues with OpenCore version 0.7.2, I also tried the debug build of v0.7.2, and release and debug builds of v0.7.3 and v0.8.2 (latest). Each time I ran into exactly the same error. Doing a Google search for the error displayed, I just found discussions with people guessing it could be a GPU issue. As my GPU ran without any problems in my older Intel Hackintosh, the hardware must be working with OpenCore; so the problem should be something else, or do you think that it's possible that a well supported GPU doesn't work in combination with this motherboard / CPU?

I'm wondering if I'm missing some important BIOS setting. In your guide you just write:
> After Updating the BIOS, you need some configuration to working.

Can you please share what changes exactly you made? The easiest way to figure this out would be in advanced view (F7), section Settings\Save & Exit
- Restore Defaults
- Save Changes and Reboot (but decline by choosing "NO")

as the save configuration dialog displays a list of which values it would revert.

**Comment by RobyRew** [comment]
ok, i'll be updating this guide next month wen I comeback to home, my friend asks me the same, i will inform you

**Comment by RobyRew** [comment]
Hey, thanks for your elaborate issue info, I want to let you know that latest BIOS that added support for 3th Gen Rizen scrued Hackintosh for Monterrey, I've been fighting trying hundreds of things for 2 continuous days but nothing with luck for Monterrey or Ventura, even downgrading to an older BIOS version.

(Big Sur works fine in all BIOS versions even the most recent one)

[These](https://dortania.github.io/OpenCore-Install-Guide/AMD/zen.html#amd-bios-settings) settings should be ok
Ill be screen capturing each BIOS Setting next time ill go to my friend's house to try and finally install Monterrey or Ventura

VERY IMPORTANT: ResizeAppleGpuBars needs to be DISABLED on BIOS that's why you have that error:
```
...
[EB|`B:WFDW] Err(0xE), 0 @ LocHB 71B4903C-14EC-42C4-BDC6-CE1449930E49
[EB|#LOG:DT] 2022-07-07T18:18:38
[EB|#LOG:EXITBS:START] 2022-07-07T18:18:38
```

I'm too sorry to respond you that late, I hope im being useful now.. 🥶

**Comment by jgehw** [comment]
Ok, this explains a lot. Originally my plan was to buy a 3950X Ryzen, but they were sold out everywhere, that's why I bought a 5950X and upgraded to the most recent BIOS.

I tried a lot of different settings and was finally able to boot and install Monterey. But whatever I tried, I couldn't get sleep to work. As working sleep is a must-have for me, I tried 5 other motherboards (Gigabyte B450, B550, X570, and X570S as well as Asus X570), but still no luck.

I then concluded that the 5950X Ryzen must be the problem. Being frustrated, I sold my used AMD hardware on ebay, and switched to an Intel-based Hackintosh (Intel 11900KF and Asus Z590 (Gigabyte Z590 didn't work)). The Intel Hackintosh is running fine and stable :-)
